maplibre-adapter/src/androidTest/java/com/amplifyframework/geo/maplibre/MapViewTestActivityTest.kt

```diff
@@ -153,15 +154,15 @@ class MapViewTestActivityTest {
                 signInWithCognito()
             }
             val result = block()
-            rule.scenario.onActivity { signOutFromCognito() }
+            rule.scenario.onActivity { }
```

Comment:
I think `signOut` was added to reset the state before running each test. Not all tests need signIn and tests are not guaranteed to run in fixed order.

Reply:
If signout is necessary before running each test it should be part of the test instead of depending on another test for the teardown. However I did check all these tests and they are running without this extra sign out. Unless you can identify a particular test that needs this?

Reply:
I thought `signOutFromCognito()` was a private function and not another test.

Reply:
Sometimes these tests fail when run on different devices.

Reply:
If I keep that command in it fails 100% of the time on our device farm so its a catch 22.